scipy/stats/tests/test_stats.py

```diff
@@ -3677,19 +3677,16 @@ def do(self, a, b, axis=None, dtype=None):
         assert_equal(x.dtype, dtype)
 
 
-class GeoMeanTestMethod(object):
-    def equal_test(self, array_like, desired, axis=None, dtype=None, decimal=7):
+class StatsTestMethod(object):
+    def equal_test(self, array_like, desired, axis=None, dtype=None, significant=7):
```

Comment:
Here and for mstats have you considered using `rtol` as an argument? I think it would be more understandable than asking for `significant` and converting it to a tolerance.

Reply:
When I changed the argument of `significant` to `rtol`, I also have concerned this issue. 
but the most of the code in test use the argument of `significant` or `decimal`' for testing a tolerance.

If I change the argument, the other parts of the test code should be changed, in my option.
How about your opinion? If it is needed, I will fix the argument.